floris/tools/optimization/scipy/yaw.py

```diff
@@ -51,9 +55,26 @@ def __init__(
                 angle (deg). Defaults to 0.0.
             maximum_yaw_angle (float, optional): Maximum constraint on yaw
                 angle (deg). Defaults to 25.0.
-            x0 (iterable, optional): The initial yaw conditions (deg). If none
-                are specified, they are set to the current yaw angles for
-                all turbines. Defaults to None.
+            yaw_angles_baseline (iterable, optional): The baseline yaw angles
+                used to calculate the initial/baseline power production in the
+                wind farm and used to normalize the cost function. If the variable
+                exclude_downstream_turbines=True then the downstream turbines
+                that are excluded in the optimization will have their yaw angles
+                fixed to their value specified in this variable; yaw_angles_baseline.
+                If none are specified, this variable is set equal to the current
+                yaw angles in the floris object that is passed to this class. 
+                Defaults to None.
+            x0 (iterable, optional): The initial guess for the optimal solution
+                of yaw angles (deg) that maximize the objective function. Note that,
+                if exclude_downstream_turbines=True, the initial guesses for any
+                downstream turbines are ignored since they are not part of the
+                optimization. Instead, the yaw angles for downstream turbines are 0.0
+                or if 0.0 is not within the specified yaw bounds, then it is equal to
+                the lower or upper bound, whichever is closest to 0.0.
```

Comment:
I think "if exclude_downstream_turbines=True, the yaw angles for downstream turbines are 0.0 or if 0.0 is not within the specified yaw bounds, then it is equal to the lower or upper bound, whichever is closest to 0.0." belongs in the description for `bnds` (here and in other functions/classes)